2-With-broker/UserDetailsClient/UserDetailsClient.Droid/Resources/Resource.Designer.cs

```diff
@@ -15,7 +15,7 @@ namespace UserDetailsClient.Droid
 {
 	
 	
-	[System.CodeDom.Compiler.GeneratedCodeAttribute("Xamarin.Android.Build.Tasks", "1.0.0.0")]
+	[global::System.CodeDom.Compiler.GeneratedCodeAttribute("Xamarin.Android.Build.Tasks", "1.0.0.0")]
```

Comment:
this file doesn't need to be included.

Reply:
ah, thanks for catching. its one of those auto update files

Reply:
yeah...i've done that a bunch of times too.